server/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+# docker-compose.yml
```

Comment:
If possible we should avoid forcing users to use docker locally :
- developers should already have a postgres database up, maybe we should create a "test" schema during initialization (like we create the default schema)
- for ci, we should move this to infra/ci